contract/src/modalities.rs

```diff
@@ -228,52 +222,9 @@ impl ToString for TokenIdentifier {
     }
 }
 
-impl ToBytes for TokenIdentifier {
```

Comment:
This is not how this should be implemented.
`TokenIdentifier` should still have `ToBytes` and `FromBytes`.
It should look more like that:
```rust
impl ToBytes for TokenIdentifier {
    fn to_bytes(&self) -> Result<Vec<u8>, bytesrepr::Error> {
        self.to_prefixed_string().to_bytes()
    }

    fn serialized_length(&self) -> usize {
        U8_SERIALIZED_LENGTH + self.to_prefixed_string().serialized_length()
    }


impl FromBytes for TokenIdentifier {
    fn from_bytes(bytes: &[u8]) -> Result<(Self, &[u8]), bytesrepr::Error> {
        let (value_string, bytes) = String::from_bytes(bytes)?;
        let value = TokenIdentifier::from_prefixed_string(value_string)?
        (value, bytes)
    }
}
```

Reply:
It's preferred to keep the `token_id` as a `String` without any prefix. Thus, let's implement `ToBytes` and `FromBytes` only if needed in the future.